### DISCUSSION TITLE
WT32-SC01-PLUS & TOROIDAL TRANSFORMER

### DISCUSSION THREAD

**Discussion by dexterbot80** [deividAlfa/stm32_soldering_iron_controller#72] [Ideas]
Hello, You have done an excellent job here.
How about the WT32-SC01-PLUS screen?
A very nice interface could be created with [this program](https://squareline.io/), for example, and we could make a station with 4 different outputs.
[https://www.youtube.com/watch?v=6uKf5Bj0xcc&t=377s](https://www.youtube.com/watch?v=6uKf5Bj0xcc&t=377s)
And I also think that if we put in a little effort, we could also integrate the vacuum pump and even the hot air blower.
Am I being too picky? What do you think? :)

**Comment by deividAlfa** [comment]
Sorry, not interested, I won't be doing anything else than minor bugfixes, this project is long done for me.

It's open source, so fork it and make a new gui using that program.
